### PR TITLE
Allow dask

### PR DESCRIPTION
**Pull request by ClaireDons**
Continuing on from work from #176, making sure tests and formatting pass. Currently sonarcloud test is failing, probably due to changes on their end (issue #182), which we plan to fix in a future pull request.

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -38,6 +38,12 @@ authors:
     given-names: Jannes
     affiliation: "Vrije Universiteit Amsterdam"
 
+    -
```
Comment (BSchilperoort) [inline CITATION.cff:41]:
```suggestion
  -
```
the other dashes are less indented. not sure if it matters

File: pyproject.toml [hunk 2]
```diff
@@ -35,6 +35,7 @@ classifiers = [
     "Programming Language :: Python :: 3.11",
 ]
 dependencies = [
+    "dask",
```
Comment (BSchilperoort) [inline pyproject.toml:38]: Is it required as dependency? The code runs fine without dask right?

**Review by BSchilperoort — APPROVED** [review]
Looks good. Just two small comments

File: pyproject.toml (hunk 2 above)
Comment (ClaireDons) [reply pyproject.toml:38]: Good point, I'll check it without and see if it still works.

Comment (ClaireDons) [reply pyproject.toml:39]: I added dask back in as the tests broke on github actions after removing, so perhaps it is needed.

**Pull request merged**